apps/api/src/app/endpoints/ai/ai.service.ts

```diff
@@ -82,24 +94,17 @@ export class AiService {
           symbol
         }) => {
           return {
-            Name: name,
-            Symbol: symbol,
-            Currency: currency,
-            'Asset Class': assetClass ?? '',
-            'Asset Sub Class': assetSubClass ?? '',
-            'Allocation in Percentage': `${(allocationInPercentage * 100).toFixed(3)}%`
+            [HOLDINGS_TABLE_COLUMNS.CURRENCY]: currency,
+            [HOLDINGS_TABLE_COLUMNS.NAME]: name,
+            [HOLDINGS_TABLE_COLUMNS.SYMBOL]: symbol,
+            [HOLDINGS_TABLE_COLUMNS.ASSET_CLASS]: assetClass ?? '',
+            [HOLDINGS_TABLE_COLUMNS.ALLOCATION_PERCENTAGE]: `${(allocationInPercentage * 100).toFixed(3)}%`,
+            [HOLDINGS_TABLE_COLUMNS.ASSET_SUB_CLASS]: assetSubClass ?? ''
           };
         }
       );
 
-    // Dynamic import to load ESM module from CommonJS context
-    // eslint-disable-next-line @typescript-eslint/no-implied-eval
-    const dynamicImport = new Function('s', 'return import(s)') as (
-      s: string
-    ) => Promise<typeof import('tablemark')>;
-    const { tablemark } = await dynamicImport('tablemark');
-
-    const holdingsTableString = tablemark(holdingsTableRows, {
+    const holdingsTableString = tablemark.default(holdingsTableRows, {
```

Comment:
This is not working on my side. However, you can revert it when everything else is finished.